### DISCUSSION TITLE
Using Multiline strings

### DISCUSSION THREAD

**Discussion by jacobtread** [ThioJoe/YT-Spammer-Purge#511] [Ideas]
You could think about using multiline strings [Reference (https://www.w3schools.com/python/gloss_python_multi_line_strings.asp)](https://www.w3schools.com/python/gloss_python_multi_line_strings.asp) instead of using the large amount of repetitive calls to println and f.write. It would really improve the readability of the whole repository also means you wont need to include all the \n chars in your print statements

**Comment by SeqBra** [comment]
PR's are open👍
Removing a bunch of prints would definitely boost performance
